test/test_overrides.py

```diff
@@ -838,5 +838,27 @@ def test_newones(self):
         n = t.new_ones((1, 2))
         self.assertEqual(type(n), SubTensor2)
 
+class TestBroadcastAllOverride(TestCase):
+    " Regression test for gh-48169"
```

Comment:
let's use the actual issue link instead of the PR link
```suggestion
    " test for gh-37141
```